airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/java/io/airbyte/cdk/testutils/ContainerFactory.java

```diff
@@ -65,10 +68,14 @@ GenericContainer<?> container() {
   }
 
   private static final ConcurrentMap<ContainerKey, ContainerOrException> SHARED_CONTAINERS = new ConcurrentHashMap<>();
+  private static final AtomicInteger containerId = new AtomicInteger(0);
 
-  private static final MdcScope.Builder TESTCONTAINER_LOG_MDC_BUILDER = new MdcScope.Builder()
-      .setLogPrefix("testcontainer")
-      .setPrefixColor(LoggingHelper.Color.RED_BACKGROUND);
+  private static final MdcScope.Builder getTestContainerLogMdcBuilder(DockerImageName imageName, List<String> methods, String realContainerId) {
+    return new MdcScope.Builder()
+        .setLogPrefix("testcontainer " + containerId.incrementAndGet() + " (" + imageName + "[" + StringUtils.join(methods, ",") + "]: "
+            + realContainerId + ") ")
```

Comment:
Prefer `String.format` or `formatted` to less readable `+` string interpolation.